[13.0][ADD] shopfloor dangerous goods

mmequignon opened the pull request:
This module introduces dangerous attributes in shopfloor

Based on:
- https://github.com/OCA/stock-logistics-workflow/pull/854

mmequignon committed:
Add shopfloor_dangerous_goods

simahawk commented on oca_dependencies.txt:
```diff
@@ -6,3 +6,4 @@ stock-logistics-workflow
 web
 rest-framework
 server-auth
+community-data-files
```
you don't need this. MQT=PIP here

simahawk commented on shopfloor_dangerous_goods/models/product_product.py:
```diff
@@ -0,0 +1,17 @@
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    is_lq_product = fields.Boolean(compute="_compute_is_lq_product")
```
I don't think this should stay here. All these fields should stay in another module.

mmequignon committed:
shopfloor_mobile: Add hooks in picking detail scenarios

mmequignon committed:
Add shopfloor_dangerous_goods_mobile

simahawk commented on shopfloor_dangerous_goods/actions/schema.py:
```diff
@@ -0,0 +1,28 @@
+        res["has_lq_products"] = {
+            "type": "boolean",
+            "nullable": False,
+            "required": True,
```
I'd say it's NOT required

simahawk commented on shopfloor_dangerous_goods/tests/test_actions_data.py:
```diff
@@ -0,0 +1,74 @@
+    def setUpClass(cls):
+        res = super().setUpClass()
+        cls._set_product_lq()
+        return res
```
return res??? there's no res here

simahawk commented on shopfloor_dangerous_goods_mobile/static/wms/.gitignore:
```diff
@@ -0,0 +1,21 @@
+.DS_Store
```
this is not needed

TDu commented on shopfloor_dangerous_goods/__manifest__.py:
```diff
@@ -0,0 +1,21 @@
+
+{
+    "name": "Shopfloor Dangerous Goods",
+    "summary": "Glue Module Between Shopfloor and l10n_eu_product_adr",
```
```suggestion
    "summary": "Glue Module Between Shopfloor and ?",
```

TDu replied:
Is it correct or maybe `stock_dangerous_goods` ?

TDu commented on shopfloor_dangerous_goods/tests/test_actions_data.py:
```diff
@@ -0,0 +1,74 @@
+# Copyright 2021 Camptocamp SA
```
That file should be imported in `__init__.py`

TDu commented on shopfloor_dangerous_goods_mobile/README.rst:
```diff
@@ -0,0 +1,79 @@
+================================
```
I wonder if it would not be better named `shopfloor_mobile_dangerous_goods`

mmequignon replied:
that was my first idea, but @simahawk disagrees

TDu replied:
ok

simahawk replied:
rationale: keep all related modules under the same "namespace". It's also handy when you list directories or browse modules in your preferred editor ;)

mmequignon committed:
Delete useless oca_dependencies.txt

simahawk reviewed (commented):
LG, can we remove the [WIP]?

jgrandguillaume reviewed (approved):
LGTM

simahawk commented:
ahhh https://github.com/OCA/stock-logistics-workflow/pull/854 I was convinced it was merge already :sweat:

simahawk committed:
shopfloor_dangerous_goods_mobile: fix css

When the product name is split on multiple lines
do not overlap with it and make sure the symbol
is displayed always at the end of the text.
